app/forms/concept_form.rb

```diff
@@ -0,0 +1,13 @@
+class ConceptForm
```

Comment:
@hweng : Yes, the form model is required by Hydra Editor so that Hydra Editor can look it up. CC has the similar pattern as well.